src/blocks/privacy.rs

```diff
@@ -0,0 +1,201 @@
+//! Privacy Monitor
+//!
+//! # Configuration
+//!
+//! Key        | Values | Default|
+//! -----------|--------|--------|
+//! `driver` | The configuration of a driver (see below). | **Required**
+//! `format`   | Format string. | <code>"{ $icon_audio \|}{ $icon_audio_sink \|}{ $icon_video \|}{ $icon_webcam \|}{ $icon_unknown \|}"</code> |
+//! `format_alt`   | Format string. | <code>"{ $icon_audio $info_audio \|}{ $icon_audio_sink $info_audio_sink \|}{ $icon_video $info_video \|}{ $icon_webcam $info_webcam \|}{ $icon_unknown $info_unknown \|}"</code> |
+//!
+//! # pipewire Options (requires the pipewire feature to be enabled)
+//!
+//! Key | Values | Required | Default
+//! ----|--------|----------|--------
+//! `name` | `vl4` | Yes | None
+//! `exclude_device` | A device to ignore, example: `["/dev/video5"]` | No | `[]`
+//! `exclude_consumer` | Processes to ignore | No | `["pipewire", "wireplumber"]`
+//!
+//! # vl4 Options
+//!
+//! Key | Values | Required | Default
+//! ----|--------|----------|--------
+//! `name` | `vl4` | Yes | None
+//! `exclude_device` | A device to ignore, example: `["/dev/video5"]` | No | `[]`
+//! `exclude_consumer` | Processes to ignore | No | `["pipewire", "wireplumber"]`
+//!
+//! # Available Format Keys
+//!
+//! Placeholder                                      | Value                                          | Type     | Unit
+//! -------------------------------------------------|------------------------------------------------|----------|-----
+//! `icon_{audio,audio_sink,video,webcam,unknown}`   | A static icon                                  | Icon     | -
+//! `info_{audio,audio_sink,video,webcam,unknown}`   | The mapping of which source are being consumed | Text     | -
+//!
+//! You can use the suffixes noted above to get the following:
+//!
+//! Suffix       | Description
+//! -------------|------------
+//! `audio`      | Captured audio (ex. Mic)
+//! `audio_sink` | Audio captured from a sink (ex. openrgb)
+//! `video`      | Video capture (ex. screen capture)
+//! `webcam`     | Webcam capture
+//! `unknown`    | Anything else
+//!
+//! # Available Actions
+//!
+//! Action          | Description                               | Default button
+//! ----------------|-------------------------------------------|---------------
+//! `toggle_format` | Toggles between `format` and `format_alt` | Left
+//!
+//! # Example
+//!
+//! ```toml
+//! [[block]]
+//! block = "privacy"
+//! merge_with_next = true
+//! [block.driver]
+//! name = "v4l"
+//!
+//! [[block]]
+//! block = "privacy"
+//! [block.driver]
+//! name = "pipewire"
+//! exclude_input = ["openrgb"]
+//! ```
+//!
+//! # Icons Used
+//! - `microphone`
+//! - `volume`
+//! - `xrandr`
```

Comment:
Reusing `xrandr` icon is convenient for screen share, but perhaps we should make it be it's own dedicated icon?

Reply:
I don't have a specific opinion in this